Libraries/CommonFunctions.psm1

```diff
@@ -199,51 +199,121 @@ Function Select-TestCases($TestXMLs, $TestCategory, $TestArea, $TestNames, $Test
     if ($ExcludeTests) {
         Write-LogInfo "$ExcludedTestsCount Test Cases have been excluded"
     }
-    return $AllLisaTests
+    return ,$AllLisaTests
```

Comment:
in order to return as collection, instead of inner object, when there's only one object in list.

Reply:
When there's only one object, isn't it still a list?

Reply:
make others functions take this return as collection. and no error occur, otherwise,  if only one object in collection. powershell will unbind to object level.  if some other function take this value and use foreach  directly, there will be error, because it's looping the nested objects from the object itself, not from a collection.

Reply:
I understand the problem you described, but for a single object, foreach should not have problem, unless the object itself is another collection or map. Is it the case here?

Reply:
yes

Reply:
I see